shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc

```diff
@@ -99,4 +106,184 @@ function drush_shoov_ci_build_release_stuck_items() {
     drush_log(dt('@delta / @count) Re-queued message ID @id after being stuck.', $params));
     $delta++;
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Release items that are marked in progress for too long.
+ */
+function drush_shoov_ci_build_release_failed_items() {
+
+  $range = 50;
+
+  // Get CI incidents of the enabled CI Builds, that haven't been fixed for more than 3 days.
+  $query = new EntityFieldQuery();
+  $result = $query
+    ->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'ci_incident')
+    ->propertyCondition('status', NODE_PUBLISHED)
+    // Incident was created more than 3 days ago.
+    ->propertyCondition('created', time() - 3*24*60*60, '<')
```

Comment:
Can we use here `strtotime`, e.g: strtotime("-3 days") ? for all the timestamps ?